packages/components/src/List/Content/Content.scss

```diff
@@ -156,7 +156,7 @@ $line-height: 30px;
 				font-weight: 400;
 				margin-right: $padding-small;
 				text-align: right;
-				text-transform: uppercase;
+				text-transform: capitalize;
```

Comment:
Why capitalize here? I heard about no text-transform? Do you have any information?